package.json

```diff
@@ -14,7 +14,9 @@
     "dev:auth": "cd microservices/auth && npm run dev",
     "dev:user-service": "cd microservices/user-service && npm run start",
     "dev:questions-service": "cd microservices/questions-service && npm run dev",
-    "dev-all": "concurrently -c \"auto\" \"npm:dev:*\""
+    "dev:rabbitMQ": "cd microservices/matching-service && npm run start:rabbitMQ",
+    "dev:matching-service": "cd microservices/matching-service && npm run dev",
+    "dev-all": "concurrently -c \"red,green,yellow,blue,magenta,cyan,gray\" \"npm:dev:*\""
```

Comment:
We can change the `install-all` npm script to use the same colours too

Reply:
concurrently was using white as one of the colours, which is not distinguishable from the log messages, so i excluded white from the list of colours

Fixed in [efc7183](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g37/pull/21/commits/efc71839b5b087bd24a8e1f3e04f623501b7be4b)